core/src/main/scala/org/bykn/bosatsu/PackageCustoms.scala

```diff
@@ -59,6 +62,39 @@ object PackageCustoms {
         pack.copy(imports = i)
     }
 
+  private type VSet = Set[(PackageName, Identifier)]
```

Comment:
Identifier should be changed to Bindable for unused names.

Reply:
We don't check that types are used. There should be a similar check for a type that is never used.